packages/grid/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+ "extends": "../../tsconfig.build.json",
+ "compilerOptions": {
+  "composite": true
+ }
```

Comment:
Missing `outDir` here like all the other components

Reply:
Done

Reply:
✅ Complete